client/starwhale/core/model/model.py

```diff
@@ -305,7 +310,30 @@ def eval_user_handler(
             )
 
     def info(self) -> t.Dict[str, t.Any]:
-        return self._get_bundle_info()
+        _manifest = self._get_bundle_info()
+        _manifest["step_spec"] = {}
+        _store = self.store
+        _om = {}
+        if _store.bundle_path.is_dir():
+            with open(
+                _store.bundle_path / "src" / DEFAULT_EVALUATION_JOBS_FNAME, "rb"
+            ) as f:
+                _om = yaml.safe_load(f)
+        else:
+            if _store.snapshot_workdir.exists():
+                with open(
+                    _store.snapshot_workdir / "src" / DEFAULT_EVALUATION_JOBS_FNAME,
+                    "rb",
+                ) as f:
+                    _om = yaml.safe_load(f)
+            elif _store.bundle_path.exists():
+                with TarFS(str(_store.bundle_path)) as tar:
+                    with tar.open("src/" + DEFAULT_EVALUATION_JOBS_FNAME) as f:
+                        _om = yaml.safe_load(f)
+            else:
+                pass
+        _manifest["step_spec"].update(_om)
```

Comment:
`_manifest["step_spec"] = _om` and remove `_manifest["step_spec"]={} in Line314`?

Reply:
done